tests/upgrade/test_upgrade.py

```diff
@@ -1,34 +1,55 @@
-import brownie
 import pytest
+import json
 
-from brownie import * 
-
+from brownie import TheVault, MyStrategy, accounts, Contract, web3, interface
 
 SETT_ADDRESS = "0xBA485b556399123261a5F9c95d413B4f93107407"
 STRAT_ADDRESS = "0x3c0989eF27e3e3fAb87a2d7C38B35880C90E63b5"
 
+NEW_REWARDS_DISTRIBUTOR = "0xa9b08B4CeEC1EF29EdEC7F9C94583270337D6416"
+USDC = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"
+
+
 @pytest.fixture
 def vault_proxy():
     return TheVault.at(SETT_ADDRESS)
 
+
 @pytest.fixture
 def strat_proxy():
     return MyStrategy.at(STRAT_ADDRESS)
 
+
 @pytest.fixture
 def proxy_admin():
     ADMIN_SLOT = "0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103"
     admin = web3.eth.getStorageAt(STRAT_ADDRESS, ADMIN_SLOT).hex()
     return Contract.from_explorer(admin)
 
+
 @pytest.fixture
 def proxy_admin_gov(proxy_admin):
     """
-        Also found at proxy_admin.owner()
+    Also found at proxy_admin.owner()
     """
     return accounts.at(proxy_admin.owner(), force=True)
 
-def test_check_storage_integrity(strat_proxy, vault_proxy, deployer, proxy_admin, proxy_admin_gov):
+
+@pytest.fixture
+def strategist(strat_proxy):
+    return accounts.at(strat_proxy.strategist(), force=True)
+
+
+@pytest.fixture
+def usdc():
```

Comment:
is this function `usdc()` locally needed?

Reply:
It is used at a few points during the upgrade test to ensure that the bribes are properly harvested and transferred to the processor.